.releaserc

```diff
@@ -0,0 +1,79 @@
+{
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        "preset": "conventionalcommits"
+      }
+    ],
+    [
+      "@semantic-release/github",
+      {
+        "assets": [
+          {
+            "path": "bin/*"
+          }
+        ]
+      }
+    ]
+  ],
+  "releaseRules": [
+    {
+      "breaking": true,
+      "release": "minor"
+    }
+  ],
```

Comment:
Leave this in if we want to ensure the version remains `v0.*` for now or remove to enable non-zero major versions.

```suggestion
```